tests/test_warc_to_zim.py

```diff
@@ -11,17 +11,21 @@
 from warc2zim.main import warc2zim
 
 
-WARCS = [
-    "example-response.warc",
-    "example-resource.warc.gz",
-    "example-revisit.warc.gz",
-    "example-utf8.warc",
-    "netpreserve-twitter.warc",
```

Comment:
can you explain why you are removing a test file? usually not a good sign 😉

Reply:
Oh, just to save space mostly :) The single-page-test.warc has one page from lesfondamentaux with favicon, so thought it would be a better test. It can also be used with the --include-domain option to include only a single domain. 

The other WARC has pages from multiple sites but no favicon, which should still work of course, but was less a primary use case..
I suppose I could keep it if you'd like.. It would need to be used with the `-a` flag to include all pages from all domains in the WARC.

Reply:
That's fine. As you didn't mention it, I wanted to make sure we're not hiding problematic tests under the carpet :)